runtime/src/main/java/io/quarkiverse/cucumber/CucumberOptions.java

```diff
@@ -40,7 +40,7 @@
     String[] features() default {};
 
     /**
-     * Package to load glue code (step definitions, hooks and plugins) from.
```

Comment:
I believe this is correct

Reply:
Yes right, I figured it out after some revisions of my english courses. It is a called `serial comma` or` Oxford comma `